statsd/test_helpers_test.go

```
@@ -0,0 +1,652 @@
+
+	finalTags := ts.getFinalTags()
+	containerID := ts.getContainerID()
+
+	return append(
+		res,
+		"_e{5,5}:hello|world"+finalTags+containerID,
+		"_sc|hello|1"+finalTags+containerID,
+	)
+}
+
+func (ts *testServer) sendBasicAggregationMetrics(client *Client) []string {
```

golangci-lint (1.62.2)

func `(*testServer).sendBasicAggregationMetrics` is unused

(unused)